mp4parse/src/tests.rs

```diff
@@ -827,7 +827,7 @@ fn skip_padding_in_stsd() {
 fn read_qt_wave_atom() {
     let esds = make_fullbox(BoxSize::Auto, b"esds", 0, |s| {
         s.B8(0x03)  // elementary stream descriptor tag
-         .B8(0x0b)  // esds length
+         .B8(0x12)  // esds length
```

Comment:
Urg... the old method doesn't care the length very much.

Now the find_descriptor() checks the length to parse each descriptor; if the descriptor length is wrong, it will return failure.